### DISCUSSION TITLE
Add CGAL to existing project in Visualo Studio

### DISCUSSION THREAD

**Discussion by esuig** [CGAL/cgal#6855] [Q&A] [answered]
Hello,

below it follows my issue in the template. Thank you in advance

## Issue Details
I'm not able to add CGAL to my existing project in Visual Studio. I have an existing project in Visual Studio (2017) and I would like to use some routines from CGAL, so I would like to be able to use it in my project. I tried but I've not been able to use the library. 
I followed the official guide (point 2) https://doc.cgal.org/latest/Manual/windows.html#title8, added the correct include folder to my project, but I get many conflicts between boost and cgal itself (for example just including <CGAL/Exact_predicates_inexact_constructions_kernel.h>). I did like this:

- downloaded CGAL package from here https://github.com/CGAL/cgal/releases (I downloaded and extracted  CGAL-5.5.zip )
- downloaded mandatory additional libraries from the same link and added to the auxiliary folder present in  CGAL-5.5 folder
- downloaded and installed Boost (1.80)
- created the right environmental variables both for Boost and Cgal and I used these to provide the include paths to my project in VS
- tried to include just <CGAL/Exact_predicates_inexact_constructions_kernel.h> and when I try to compile I get many conflicts between boost and CGAL (e.g. a constant declaration in cgal is recognised as variable name in boost and things like this)

## Environment
* Operating system (Windows 64 bits):
* Compiler: I'm using visual studio 2017
* Release or debug mode: Release
* CGAL version: 5.5
* Boost version: 1.80
* Other libraries versions if used (Eigen, TBB, etc.): no additional libraries used

**Comment by janetournois** [comment, marked as answer]
The easiest way is to use CMake as explained [here](https://doc.cgal.org/latest/Manual/devman_create_and_use_a_cmakelist.html) in the documentation.
